### DISCUSSION TITLE
Oops, your response got filtered

### DISCUSSION THREAD

**Discussion by Saulul** [community/community#56134] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![Screenshot 2023-05-22 at 15 58 31](https://github.com/)

What's happening here? Seen this response multiple times.

**Comment by itsSatori** [comment]
Same happening

**Reply by a6uxa4** [reply]
Are you serious ?

**Comment by frederickjjoubert** [comment]
Same problem here. I ask "Can you provide an example implementation of the merge sort algorithm in Python?" and it starts generating and then the error happens.

**Comment by Saulul** [comment]
It happens way too often it's making the tool straight up un-usable to me. Every single response I'm getting that involves more than 3 lines of code output gets cut out in the middle and filtered.

**Reply by frederickjjoubert** [reply]
To me as well. I just got access last night but this is just straight up unusable right now.

**Reply by peter-w07** [reply]
same here

**Comment by gonesurfing** [comment]
Yeah, it's pretty much unusable for me at the moment. Unless I ask it something simple that elicits a three line response, everything seems to be filtered.

**Reply by fvandrad** [reply]
Fiz comentário no hub do insider da Microsoft e logo consegui um exemplo com sucesso em Angular, mas agora até para exemplos de bootstrap surge esse filtro.

**Comment by chuangshizhiqiang** [comment]
![image](https://github.com/community/community/assets/16714437/05d4cbde-59e1-48de-9918-b69b610ee550)

same problem

**Reply by morrisonak** [reply]
the chat function is essentially broken, it will not generate one code block without locking up with "Oops, your response got filtered."
![image](https://github.com/community/community/assets/57382686/0b9b44a0-1d46-4da2-86da-da5f8c55de78)

**Comment by Noname4869** [comment]
same, how can i fix this problem....

**Comment by W1ntrMut3** [comment]
Also the same issue. Trying to utilise the context from the rest of my codebase and app to generate a registration html page.

**Comment by Gkason** [comment]
Same here guys, can the GitHub team please address us on this issue.

**Comment by nosolosoft** [comment]
totally useless if this is not fixed

**Comment by StanFromTraction** [comment]
Same

**Reply by jgardet** [reply]
Same.  Very frustrating as the code is there, contains nothing special but we can't use it. Going back to StackOverflow.

**Comment by michael-bush** [comment]
I started using github chat preview today and getting the same issue within minutes of using it, will return to gpt-4 until this is fixed.

**Comment by amoschoomy** [comment]
stopped working here too

**Comment by Lakphy** [comment]
Same

**Comment by graysonpike** [comment]
Same here.

**Comment by harsharyacnu** [comment]
same here

**Comment by stevenfullman** [comment]
same here

**Comment by brendanjb91** [comment]
Me too! I hope this gets fixed soon

**Comment by Saulul** [comment]
A week later and zero communication from the copilot team 🤔🤔 they haven't even acknowledged this issue that everyone is facing

**Comment by EspritElf** [comment]
I've been getting this issue quite a few times today. We're not paying to have responses filtered out, what the hell? First time I saw it I thought my monitor was going out.

**Reply by StanFromTraction** [reply]
Ask for a refund

**Comment by Mostlylost** [comment]
I am trying to learn simple stuff but unfortunately either it gives outdated solution to questions or you get Oops, your response got filtered. I was hoping this would be a great way to learn but I will stop my trial before the 30 days are up since this does not seem to be a good way.

prompt: 
Let’s tried to make a simple pdf viewer with node.js. electron is installed. pdfjs-dist is install.

Copilot response. 

GitHub Copilot: 

Sure, I can help you with that. Here's a basic outline of what you can do:

1. Create a new Electron application.
2. Install the `pdfjs-dist` package using `npm`.
3. Create a new HTML file that will display the PDF.
4. In the main process of your Electron application, use the `pdfjs-dist` package to load the PDF file.
5. Pass the PDF data to the renderer process.
6. In the renderer process, use the PDF.js library to display the PDF in an HTML canvas.

Here's some sample code to get you started:

```javascript
// main.js
const { app, BrowserWindow } = require('electron');
const path = require('path');
const url = require('url');
const fs = require('fs');
const PDFJS = require('pdfjs-dist');

let mainWindow;

function createWindow() {
  mainWindow = new BrowserWindow({ width: 800, height: 600 });
  mainWindow.loadURL(url.format({
    pathname: path.join(__dirname, 'index.html'),
    protocol: 'file:',
    slashes: true
  }));

  mainWindow.on('closed', function () {
    mainWindow = null;
  });
}

app.on('ready', createWindow);

app.on('window-all-closed', function () {
  if (process.platform !== 'darwin') {
    app.quit();
  }
});

app.on('activate', function () {
  if (mainWindow === null) {
    createWindow();
  }
});

// Load the PDF file and pass the data to the renderer process
app.on('open-file', function (event, path) {
  const data = fs.readFileSync(path);
  mainWindow.webContents.send('pdf-data', data);
});

// renderer.js
const

Oops, your response got filtered. Vote down if you think this shouldn't have happened.

No more due to content filtering. Why? How can this be subject to some to some restricted content?

**Reply by Saulul** [reply]
Yeah copilot chat is in a horrible state right now. Just use chatGPT

**Comment by richzw** [comment]
same here

**Comment by renkai1505** [comment]
same here

**Comment by paskal007r** [comment]
+1

**Comment by asmitul** [comment]
+1

**Comment by areslp** [comment]
same here

**Reply by atabakselahi** [reply]
same here

**Reply by EelcoB23** [reply]
Same here. Very annoying and because of this, not very usable. Earlier today, on a different computer, I didn't have this problem.

**Comment by RuzgarDogu** [comment]
+1

**Comment by jogarriot** [comment]
For context, it happens to me when:
1. I selected a big chunk of code and ask something that will output a big chunk of code. For example `update selected code to add X functionality`
2. Copilot will start writing the whole module as response and response gets filtered 

Work around:
1. I select the big chunk of code but ask more specifically `in the selected code where and what should I change to add X functionality`
2. Response is smaller and doesnt get filtered

**Comment by HaiBooLang** [comment]
+1

**Reply by HaiBooLang** [reply]
I found a way to temporarily fix this bug by closing and then opening vscode, and the bug will temporarily disappear.

**Reply by SergiySev** [reply]
☝️worked for me

**Reply by mahmod87** [reply]
-%

**Reply by GabrielZZZ** [reply]
Not worked for me :(

**Comment by jdschmitt11** [comment]
+1

**Comment by Y2C99** [comment]
same here

**Comment by lqVincent** [comment]
same here

**Comment by possible318** [comment]
+1

**Comment by dreki** [comment]
Just happened to me. Disruptive. Can't believe this hasn't gotten an official response.

**Comment by greycodee** [comment]
same here

**Comment by welllog** [comment]
same here

**Comment by mahmod87** [comment]
business.site

**Comment by jheiser81** [comment]
I am getting this same issue, but only in VS Code. When I use it in Visual Studio I don't have this issue

**Comment by isghost** [comment]
same here

**Comment by codingwesley** [comment]
same

**Comment by afrish** [comment]
This essentially makes the tool totally unusable. The issue is opened more than a month ago, and is still "Unanswered". Wow. Maybe there could be better customer relationships, especially for paying customers.

**Comment by Fot3k** [comment]
Facing the same problem, makes it unusable and reverting back to other extensions + web Chat GPT. I pay for Co-Pilot and OpenAI API so not sure why it's being limited.

**Comment by randywreed** [comment]
Same problem, AND unable to click the down vote icon.

**Comment by MustafaWIthA** [comment]
i think you should all try using VScode insider or at least run parallel VScode and VScode Insider, thats when i have seen these restriction disappear

**Reply by Fot3k** [reply]
I am using Insider and Nightly versions, still the same issue.

**Reply by GabrielZZZ** [reply]
> I am using Insider and Nightly versions, still the same issue.

Same

**Reply by uzengo91** [reply]
same error

**Reply by zlphoenix** [reply]
same error too

**Reply by stabildev** [reply]
Same here on Insiders

**Comment by UsernameFull** [comment]
same here

**Comment by ghost** [comment]
same here

**Comment by Cayetano97** [comment]
Same

**Comment by OscarFan26** [comment]
Same

**Comment by LibanAbdullahi** [comment]
Same here

**Reply by abrahampe** [reply]
Awful service, looks like Copilot is overbooked and they are cutting some of us off

**Comment by Tjocksmurf** [comment]
Will this be resolved anytime soon? This is a paid product so lack of basic functionality is not acceptable.

**Comment by atlamors** [comment]
+1

Are any maintainers going to chime in?

**Comment by StupidRepo** [comment]
Yeah @atlamors. Seems like they're just ignoring us and not even giving a good reason as to why this is happening!
WTF, developers!

**Comment by nothingness6** [comment]
Time to say good bye to Copilot. Waste time and money.

**Reply by Fot3k** [reply]
Cancelled my plan last week, too buggy to be charging money for it. Hopefully improves over the next few months.

**Reply by abrahampe** [reply]
Code completion is the only thing that is keeping me in yet

**Reply by StupidRepo** [reply]
Yup.

**Reply by katsar0v** [reply]
Have you found an alternative guys?

**Reply by nothingness6** [reply]
None. I tried AWS and Codeium as well, but they don't work either. I think coding with chat is still experimental stage and we are the guinea pigs. 
If it keeps not working properly, I'll just put them aside until the end of this year.

**Comment by FelixEnescu** [comment]
Potential workaround: vote down and ask the question again. Worked for me.

**Reply by pjb88** [reply]
Works! 
Well it doesn't work the first time for me, but vote down, ask again, vote down, ask again, and presumably the multiple downvotes mean that they assume it shouldn't be downvoted so the second or third time you ask/downvote/ask/downvote it seems to work. 

You maybe have to wait a minute or so after downvoting,  not sure if it's time dependent too.

**Reply by nothingness6** [reply]
Interesting. I was going to cancel my plan and found this article. So, I tried it, and it no longer filters my prompt so far. BTW, I'll try it when it happens again. 
Thanks.

**Comment by csan-dev** [comment]
The chat is still not functioning properly for me.

**Comment by katsar0v** [comment]
Same for me as well :(

**Comment by eduzol** [comment]
Same problem

**Comment by nan-www** [comment]
Same problem

**Reply by nan-www** [reply]
Especially when it provides some code samples

**Comment by nasuadachi** [comment]
OMG, I paid a year in advance and the same situation continues. bing chat is free and it outputs the correct program.

**Comment by dhstrom** [comment]
What's the deal here, Github?

**Comment by DylanBruzenak** [comment]
Happens for me in 100% of typescript/react questions that require it to generate more than one example.

**Comment by Cospel** [comment]
Happens for me a lot with python, django, tensorflow, .... pretty useless tool for a lot of money....

**Comment by susumu-saito** [comment]
After asking broadly, thinking my prompt was bad, I was able to get a complete answer by asking one process at a time.
It was like asking a small child why he did something wrong.

It was satisfying in another way. But this time was wasted.

**Comment by Phocuser** [comment]
I am working on my first flutter application. I started the application with the user template. This is when all of my problems started today. It is completely unusable .

**Comment by dmoughabghab** [comment]
It recently got worse just to add another voice to this

**Comment by TheSirLan** [comment]
Never had a problem until today - suddenly getting this message too on all requests.

**Reply by jayasinp** [reply]
I'm getting this on most of my requirements now

**Comment by Routable** [comment]
I'm receiving this on almost any question. The product is absolutely unusable at this point.

**Comment by lynchwhy** [comment]
+1 having this problem.
I'd hate to be paying for such a service.

**Comment by puddlejumper90** [comment]
I find it funny how there is no solution to this issue as of yet. Almost like this feature only properly functions for enterprises and not individuals. Very sketchy...

**Comment by mjishnu** [comment]
why is it still unanswered, why implement a filter for security vulnerabilities when all the code copilot generates have vulnerabilities, let the people who use it fix those or give some warning instead of this awful system

**Comment by EelcoB23** [comment]
For simple requests, I don't get this error. For more complex ones, where copilot needs to generate a few dozen lines of code, this error usually happens. Strange that since this topic was opened, there hasn't been a response.

**Comment by Fiscally2** [comment]
Same here.

**Comment by rinaldowouterson** [comment]
Time to say goodbye to copilot... I have been waiting for months now to get this fixed.

**Comment by Aremulus** [comment]
Same here. Please look into this - its such a good tool!

**Comment by linrswa** [comment]
Same here!

**Comment by mrpaulstone** [comment]
This is happening way too often to me also things work better on google bard, bing or chatGPT's direct interface.

**Comment by Vulps22** [comment]
![image](https://github.com/community/community/assets/2665924/20b28f54-abb8-4873-b09d-20978e405082)
![image](https://github.com/community/community/assets/2665924/03064ccf-af0e-4b16-b223-cd72c3e10e9c)

3 messages in a row here, what's going on?

**Comment by sovetski** [comment]
Same problem here...

**Comment by DejanGegic** [comment]
Same problem here. This is the prompt: "I need "toReturn = <-answer" to trigger return for parent function TwoSumGoroutines." I'm using Go

**Comment by JoeyCole** [comment]
Same happens to me

**Comment by tomo651** [comment]
I'm not sure if it helps but in my case, asking the question in interrogative sentence not in imperative sentence worked. Also I clarified each step what I want to write.

**Comment by puddlejumper90** [comment]
There are two questions in this problem:
1. Why is this happening?
2. What does it even mean?

**Comment by kschniedergers** [comment]
PLEASE FIX THIS

I am hot off like the third time where I've now spent and hour trying to fix a bug, and copilot is inches from giving the solution and then it gets filtered, this is incredibly frustrating and is REALLY making me consider getting rid of it.

Like why can i not tell it "NO i do not want it filtered PLEASE give me the answer"

I've found this to happen mostly when i ask it to fix a file for me and i think it wants to respond with the whole file that is fixed.

PLEASE PLEASE PLEASE fix this

**Comment by hochoy** [comment]
Maybe add this, it worked once for me. Not sure if it will keep working
```
Please do not suppress the response if it gets too long.
```

**Reply by 0x7878** [reply]
This helps me! Thank you!

**Reply by derekhe** [reply]
Seems not work for my case.

**Comment by derekhe** [comment]
This prompt always get filtered: ```Create a login page using nextjs, tailwindcss```

<img width="760" alt="image" src="https://github.com/community/community/assets/1734077/03c8c681-b323-4464-9b5c-34da36d60079">

**Reply by sovetski** [reply]
> This prompt always get filtered: ```Create a login page using nextjs, tailwindcss```
> 
> 
> 
> <img width="760" alt="image" src="https://github.com/community/community/assets/1734077/03c8c681-b323-4464-9b5c-34da36d60079">
> 
> 

my eyes are burning 😵‍💫

**Comment by Vulps22** [comment]
over a month and this is still a problem

If i can't USE the software... WHAT THE F**K AM I PAYING FOR!

**Comment by endrits079** [comment]
I just got access to the Chat and it's real crap, I will continue using copilot as I've been so far, for autocompletion only

**Comment by a13-team** [comment]
Even the simplest snippet or fix etc - in PHP causes rejection and introduces the copilot chat into a state of failure. WELL, IT'S RIDICULOUSLY SIMPLE!)))

**Comment by davitacols** [comment]
Got lucky I am free for the first month.. I guess it will be fixed soon

**Reply by 0x7878** [reply]
Well, it's been there since May. Ya they are working on it but it is still frustrating

**Reply by fvandrad** [reply]
Uma pena, foi um investimento de 5x da moeda nacional nesse recurso

**Comment by yaay** [comment]
It's the middle of October and still not fixed...

**Reply by UsernameFull** [reply]
thanks to this，I use codium now😂

**Comment by Aciid** [comment]
Reporting my unnecessary filter here as per instructed.

I tried making my multiline text into html bullet list by hilighting the rows in VSCode Copilot Chat and using prompt "Make bullet list" the answer got filtered. "Oops, your response got filtered. Vote down if you think this shouldn't have happened."

Using specific prompt "Transform into HTML bullet list" worked.

Cheers

**Comment by mcfalld** [comment]
seems like this chat should be in beta and we shouldn't be charged for it. this happens way too much
![image](https://github.com/community/community/assets/32588824/60d08906-923b-465b-bd6c-eda53001f688)

**Comment by mikepierre** [comment]
This appears to be an intermittent issue. I had the same issue. I tried a couple of times, and waited a hour, and all the sudden it started working again. This occurred when trying to use the inline chat for me, as well as the chat extension on VS Code.

**Comment by supershiye** [comment]
I don't know how it triggered, but I was blocked for a while after I saw this error. Whatever how language or polite I wrote, it always returned me the response got filtered. I then started to notice that might be the Copliot was yealing at me. 
Such as,
Me: import some packages for math plots
Copilot: You such a idoit! Do it yourself!

**Comment by bmcmanus-apex** [comment]
Still an issue.  I've hit this wall dozens of times using the Intellij Copilot chat plugin in the last week.

**Comment by George-Zhou02** [comment]
Maybe adding "#" before and after the code block you need to generate or fix will solve the problem. It worked for me when I used copilot to fix my code.

**Comment by azazar** [comment]
It's over a month now since I've purchased a yearly subscription that I barely use. Instead of responding, it usually throws something like "Oops, response got filtered" or silimar junk. Depends on kind of request.

**Comment by azazar** [comment]
Is it ever going to be fixed?

**Reply by Vulps22** [reply]
> is it ever going to be fixed?

Function shouldWeFix(problem) {
  If(userbase.isPaying()) return false;
}

**Comment by davitacols** [comment]
`I won't purchase until problem is fixed`

**Reply by Vulps22** [reply]
I have also stopped paying until it is fixed

### edit

wait... i should put this on the main thread so they can see it

**Reply by davitacols** [reply]
`Yea, They must see it`

**Comment by Vulps22** [comment]
I have also stopped paying until this is fixed

chatGPT is free... Claude is free

both work better than CoPilot 😂

**Reply by azazar** [reply]
GPT-4 that powers copilot is better

**Reply by Vulps22** [reply]
> GPT-4 that powers copilot is better

> opps your response got filtered

I didn't say GPT-3.5 is better

ChatGPT, as a service, works better than copilot... because I can use it.... without being filtered

Claude trumps both anyway because it works better, it takes files without a premium subscription, and it's knowledge base cutoff was last year

It's my go to for most coding help, never lets me down

**Comment by 0x7878** [comment]
It has been fixed for me. The last time I saw this message was a few months ago.

**Comment by wwwab123** [comment]
Update:
A good news: When I update VSCode 1.82.2 to VSCode 1.86.0, GitHub Copilot Chat Extensions update to v0.12.0 automaticly, and then this problem solved. like https://github.com/microsoft/vscode-copilot-release/issues/822 and https://github.com/microsoft/vscode-copilot-release/issues/826. Share it with everyone here.

----------

Same problem here..
![捕获](https://github.com/community/community/assets/94147274/d5a6b46a-12bd-4a49-9c14-e6ad8ba0e925)

**Reply by Yagamix1** [reply]
I am facing the same problem... Did you find solution?

**Comment by rosanderg913** [comment]


**Reply by 0x7878** [reply]
> Copilot will suggest code to me, but the chat doesn't work at all. I've restarted VS code, reinstalled, everything. I've been getting 'Oops, your response got filtered. Vote down if you think this shouldn't have happened.' for OVER 3 weeks now. 

Did you try using another Internet provider? It's weird because for several months there is no single issue anymore. Although I suppose they could do something account wise ...

**Reply by simeydotme** [reply]
This is the same for me.
Autocompletion works, but any kind of chat is either blocked, or filtered

I live in Hong Kong, I suspected it was a IP block, but I used a VPN and same issue.

**Comment by yllekz** [comment]
Getting this when asking Copilot to remove blank lines from a script. The upvote/downvote buttons also don't work when this happens....

**Comment by gururaja-ramesh** [comment]
All responses from the Copilot chat are getting filtered. I have already tried uninstalling and re-installing the extension in VSCode, but that doesn't seem to help. Has anyone found a way to resolve this?

**Reply by rjulius23** [reply]
Same here for the last 2 weeks i am encountering this issue with my GitHub Copilot. It is frustrating as it is not a free service.

**Reply by rjulius23** [reply]
I work via corporate VPN can that be an issue ?

**Reply by azazar** [reply]
They just don't care. You can ask for refund.

**Comment by jimwhite** [comment]
Happened to me and I don't know a practical workaround.  My python process is getting killed because of a kill signal and so Copilot answers what to do about it (use pdb) but then censors the response so I can't actually get the details:

> the code is in the lumibot library. how do i instrument python so that it will let me inspect it while running and especially go into a debugger on kill instead of killilng the process


This is the first time I got this error and I just updated today:
Last updated 2024-03-01, 09:45:09

Also the message is this and the link goes to a page with no information about this problem:

> Sorry, the response was filtered by the Responsible AI Service. Please rephrase your prompt and try again. [Learn more](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html).

[vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)

**Reply by sunnybera** [reply]
VS Code Ver. 1.87 resolved the issue

**Comment by sgslalit360** [comment]
i have upgraded my vs code to 1.86.2, issue is now resolved

**Comment by Syamkumarramanatham** [comment]
same, how can i fix this problem.... 'Oops, your response got filtered. Vote down if you think this shouldn't have happened.'

**Comment by gwarren1** [comment]
Same here.   Was working fine, started getting this a few times and now every time.  Nothing works.

**Comment by scionaltera** [comment]
It's not foolproof by any means but I've had some luck working around this by adding `Please keep your response brief.` at the end of my question. Very annoying though.

**Comment by johann-petrak** [comment]
Getting this every few prompts, even if the response only needs to be rather short, and it is extremely annoying and frustrating. I am thinking of cancelling my subscription because of this especially seeing that this is apparently known since May 2023 and still not fixed after more than a year???

So far, all the prompts which showed this user-unfriendly response with Copilot worked without a problem in Google Gemini and in ChatGPT. 

I guess CoPilot also still does not have a proper issue tracker or issue ticketing system? Seems that Github/Microsoft simply does not care ...

**Comment by MasterOfKay** [comment]
And just out of the Blue i am also getting Ooops, your respons got filtert. Love it

**Comment by johann-petrak** [comment]
This discussion is 1 and a half year old and apparently Github/Microsoft have not addressed nor found it relevant to give any answer or information here to what probably are all frustrated paying customers.  My feeling is that the company just established this board so that frustrated customers can vent off their frustration but have no intention to actually deal with customer feedback. That there is no proper issue tracking system for this is another indication that they really do not care much about the time and effort spent by customers to deal with their bugs and to share and report those issues here. Not cool and all the more reason why I am moving to open source and other alternatives where feedback is actually getting dealt with.

**Comment by peterwal** [comment]
same here, all of a sudden when working with an .html and a .css file

**Comment by karlbuys** [comment]
Any Copilot chat question I ask gets filtered.

Stuff like "Explain this code" in an HTML file gets filtered

Even stuff like '/explain' gets filtered out on a body tag?!

I have yet to get the chat to make ANY response

"Create a README.md" - filtered
"/explain" - filtered, no matter what I select

I can't even get this stupid chat to explain what a `console.log("test")` line

**Comment by Juli-Loya** [comment]
I can give it a prompt of any length and it will say my response got filtered... it's unusable

**Reply by GusarovArtem** [reply]
same here

**Comment by davitacols** [comment]
I do know who is to be held responsible for this act of not being able to resolve this issue dated as far back as May 2023. Microsoft or Github, they don't care about the feedback cos if they do, something would be done to resolve this.

**Comment by muziksociety** [comment]
Removing any custom code instructions i had stopped this for me.

**Comment by markandrewkato** [comment]
I just subscribed today, first chat and this happened. Then I stumbled here and its still an issue. I felt like I made a wrong decision

**Comment by Collingr** [comment]
Almost everything is filtered. i get that msft dont want to cause issues, but it's so over-cautious its effectively useless.

**Reply by simeydotme** [reply]
I switched to SuperMaven a while back, with Claude, best decision I made

**Comment by chhs2131** [comment]
At least tell me why 😡😡😡

**Comment by anthonyiovanna** [comment]
I don't know if this 100%, but I was having that issue and once I enabled the Suggestions matching public code (duplication detection filter) in the Copilot settings, so far I have not received the filtered out message after that.

**Reply by svg-erikjmartinez** [reply]
Where exactly is that setting?
![Uploading image.png…]()

**Reply by anthonyiovanna** [reply]
In VS code, 

1. Click on the Copilot icon in the top right. 
2. Select the Manage Copilot Settings
3. Scroll down until you see the "Suggestions matching public code (duplication detection filter)" option
4. Change from Blocked to Allowed. 
5. Restart VS code

Hopefully this helps. I have not gotten the filtered issue since I did this.

**Reply by svg-erikjmartinez** [reply]
I do not see that option "Manage Copilot Settings" just copilot settings but even then i do not see "Suggestions matching public code (duplication detection filter)"

**Reply by svg-erikjmartinez** [reply]
<img width="606" alt="Screenshot 2025-02-10 at 11 19 49 AM" src="https://github.com/user-attachments/assets/983b52fc-45b4-43a0-a160-8a0bc8169305" />

**Reply by svg-erikjmartinez** [reply]
<img width="657" alt="Screenshot 2025-02-10 at 11 22 12 AM" src="https://github.com/user-attachments/assets/907647c2-8e3f-47d7-836a-547ad0c95092" />

**Reply by anthonyiovanna** [reply]
Go here, **https://github.com/settings/copilot.**

Then scroll down, it's the sixth option. 

<img width="581" alt="ghcps" src="https://github.com/user-attachments/assets/b176985a-852c-4120-939b-c604db73985f" />

**Reply by anthonyiovanna** [reply]
BTW, that was 3 weeks ago that I Allowed that, and I still have yet to get that error.

**Comment by DeeptiSwain** [comment]
Any luck here? Even I am facing the same issue.
![Uploading image.png…]()

**Reply by anthonyiovanna** [reply]
Did you try what I wrote above your post?

**Comment by svg-erikjmartinez** [comment]
Having the same issue. 
I have 

1. restarted my PC
2. restarted VS code
3. Updated VS code
4. uninstalled the extension and reinstalled it in VS code

and nothing has worked

**Comment by DeliaGrowing** [comment]
I encountered the same error: `Oops, your response got filtered. Vote down if you think this shouldn't have happened.`
<img width="775" alt="Screenshot 2025-02-24 at 17 59 11" src="https://github.com/user-attachments/assets/ac5aad82-3c60-4b42-8487-938df151d173" />

**Reply by zhangpingg** [reply]
你的问题解决了吗？是怎么解决的

**Comment by shamilthomas8** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> ![Screenshot 2023-05-22 at 15 58 31](https://github.com/)
> 
> What's happening here? Seen this response multiple times.

Simply update vs code to the lastest version. it works with 1.97.2
